linera-execution/src/lib.rs

```diff
@@ -1303,7 +1305,7 @@ pub struct BlobState {
 
 /// The runtime to use for running the application.
 #[derive(Clone, Copy, Display)]
-#[cfg_attr(with_wasm_runtime, derive(Debug, Default))]
+#[cfg_attr(any(with_wasmer, with_wasmtime, with_revm), derive(Debug, Default))]
 pub enum WasmRuntime {
```

Comment:
VmRuntime?

Reply:
I actually find it better not to add Revm to `WasmRuntime` and so no renaming is needed:
* Having Wasmer / Wasmtime / Revm seems wrong. We should have a top enum `Wasm`, `Evm`, `Solana`. `Wasm`should itself be an enum that contains `Wasmer` / `Wasmtime`. `Evm` should have just `Revm` right now, and `Solana` could maybe have `Wasmer`/ `Wasmtime` (will see later).
* I do not need to add the `Revm` to the `Wasmtime` in order to make the test pass.